__tests__/textalyze.test.js

```diff
@@ -87,6 +87,12 @@ describe('itemFrequencies', () => {
 });
 
 describe('getPrintStatistics', () => {
+  test('throws an error when the map isn\'t a valid one', () => {
+    expect(() => {
+      getPrintStatistics(null);
+    }).toThrow(new TypeError('The passed map must be a valid one.'));
```

Comment:
This test isn't bad, but in my opinion, testing

```javascript
.toThrow(new TypeError('The passed map must be a valid one.'))
```

makes the test very brittle.  If you ever change the error message, the tests will break.  I'd check for only the type of error being thrown, e.g., `.toThrow(TypeError)`.

If that doesn't feel specific enough to you then I'd create a custom error class that represents your error more specifically, e.g., `InvalidMapError`.  You could make it a subclass of `TypeError` if you want.